### ISSUE TITLE
Retrieve price from other supermarkets

### ISSUE THREAD

**Issue by Daniel-ltw** [Daniel-ltw/supermarket-deals-ror#122]
Currently workable supermarket is pak and save. 

Work has to be done, scraping the site and seeking out the right product for the product id and adding prices object with that product id.

**Comment by nicholasold** [comment]
Might be interesting to have another look and see if anyone has a public API?  I tried New World just now though and it's crap - you'd have to parse a PDF :(

**Comment by Daniel-ltw** [comment]
@nicholasold Yup, I do not think they will be willing to offer an api to scrutinise themselves.

**Comment by thomashorrobin** [comment]
Are there any inadvertently public APIs? I'm thinking like the Metlink APIs that are open to the world by accident/misfortune (https://www.metlink.org.nz/api/v1/StopDepartures/4323). If you guys don't know of any I might have a hunt around.

**Comment by Daniel-ltw** [comment]
Feel free to give it a try. 

Based on my understanding, most of them do not.
